scripts/ci/run_perftest.sh

```diff
@@ -315,35 +269,100 @@ org_flag() {
   esac
 }
 
-# Generate a DBRP mapping for use by InfluxQL queries.
+create_dbrp() {
 curl -XPOST -H "Authorization: Token ${TEST_TOKEN}" \
   -d "{\"org\":\"${TEST_ORG}\",\"bucketID\":\"$(bucket_id)\",\"database\":\"$db_name\",\"retention_policy\":\"autogen\"}" \
   http://${NGINX_HOST}:8086/api/v2/dbrps
+}
+
+##########################
+## Run and record tests ##
+##########################
+
+# Generate queries to test.
+query_files=""
+for usecase in window-agg group-agg bare-agg group-window-transpose iot metaquery multi-measurement; do
+  for type in $(query_types $usecase); do
+    query_fname="${TEST_FORMAT}_${usecase}_${type}"
+    $GOPATH/bin/bulk_query_gen \
+        -use-case=$usecase \
+        -query-type=$type \
+        -format=influx-${TEST_FORMAT} \
+        -timestamp-start=$(start_time $usecase $type) \
+        -timestamp-end=$(end_time $usecase $type) \
+        -queries=$queries \
+        -scale-var=$scale_var > \
+      ${DATASET_DIR}/$query_fname
+    query_files="$query_files $query_fname"
+  done
+done
+
+# Generate and ingest bulk data. Record the time spent as an ingest test if
+# specified, and run the query performance tests for each dataset.
+for usecase in iot metaquery multi-measurement; do
+  data_fname="influx-bulk-records-usecase-$usecase"
+  $GOPATH/bin/bulk_data_gen \
```

Comment:
Since there's been a lot of talk on Windows test errors, figured I'd ask: it's fine to have hardcoded Unix-paths here (and elsewhere) right since it will always run on a Linux machine when running these perf tests?

Reply:
Yeah we're currently only running them on a specific Linux distro. Running them on different OS's would be interesting and possibly useful, but would require a pretty significant overhaul.